mithril-signer/src/message_adapters/from_pending_certificate_message.rs

```diff
@@ -61,18 +19,10 @@ impl TryFromMessageAdapter<CertificatePendingMessage, CertificatePending>
             signed_entity_type: message.signed_entity_type,
             protocol_parameters: message.protocol_parameters,
             next_protocol_parameters: message.next_protocol_parameters,
-            signers: to_signers(&message.signers).with_context(|| {
-                format!(
-                    "'FromPendingCertificateMessageAdapter' can not convert the list of current signers: '{:?}'",
-                    message.signers
-                )
-            })?,
-            next_signers: to_signers(&message.next_signers).with_context(|| {
-                format!(
-                    "'FromPendingCertificateMessageAdapter' can not convert the list of next signers: '{:?}'",
-                    message.next_signers
-                )
-            })?,
+            // This field is deprecated and should not be used in Signer.
```

Comment:
Same typo for this line and the line below:
```suggestion
            // This field is deprecated and should not be used by the Mithril signer.
```

Reply:
Done